pioneer/packages/joy-roles/src/transport.substrate.ts

```diff
@@ -510,9 +458,9 @@ export class Transport extends TransportBase implements ITransport {
         maxNumberOfApplications: opening.max_applicants,
         requiredApplicationStake: stakes.application,
         requiredRoleStake: stakes.role,
-        defactoMinimumStake: new u128(0)
+        defactoMinimumStake: this.api.createType('u128', 0)
```

Comment:
Balance instead of u128 ?

Reply:
Addressed in https://github.com/Joystream/joystream/pull/1195/commits/728bb4c90c228828a77325f387efa9787f720711